Source/CarthageKit/Xcode.swift

```diff
@@ -1342,6 +1348,20 @@ public func architecturesInFramework(frameworkURL: NSURL) -> SignalProducer<Stri
 		}
 }
 
+/// Strips `Modules` directory from the given framework.
+public func stripModulesDirectory(frameworkURL: NSURL) -> SignalProducer<(), CarthageError> {
+	return SignalProducer.try {
+		let modulesDirectoryURL = frameworkURL.URLByAppendingPathComponent("Modules", isDirectory: true)
+
+		var error: NSError? = nil
+		if !NSFileManager.defaultManager().removeItemAtURL(modulesDirectoryURL, error: &error) {
```

Comment:
Should you check if the file exists? Not all frameworks will include `Modules`, right?